libraries/azure_vpn_gateways.rb

```diff
@@ -0,0 +1,46 @@
+require 'azure_generic_resources'
+
+class AzureVPNGateways < AzureGenericResources
+  name 'azure_vpn_gateways'
+  desc 'Verifies settings for AKS VPN Gateways'
```

Comment:
Looks like there is a typo here: Azure Kubernetes Service (AKS).

Reply:
I've changed this now, thanks for spotting this.